src/effects/effectslot.cpp

```diff
@@ -31,9 +31,10 @@ EffectSlot::EffectSlot(const QString& group,
 
     m_pControlEnabled = new ControlPushButton(ConfigKey(m_group, "enabled"));
     m_pControlEnabled->setButtonMode(ControlPushButton::POWERWINDOW);
-    // Default to enabled. The skin might not show these buttons.
-    m_pControlEnabled->setDefaultValue(true);
-    m_pControlEnabled->set(true);
+    // Default to disabled to prevent accidental activation of effects
+    // at the beginning of a set.
+    m_pControlEnabled->setDefaultValue(false);
```

Comment:
These are both no-ops since they're the default. Maybe drop them and move the comment above the "new ControlPushButton".